packages/node-client/lib/index.ts

```diff
@@ -30,10 +30,12 @@ export class Nango {
      *
      * @remarks
      * For OAuth 2: returns the access token directly as a string.
+     * For OAuth 2: If you want to obtain a new refresh token from the provider before the current token has expired,
+     * you can set the instantRefresh argument to true."
      * For OAuth 1: returns an object with 'oAuthToken' and 'oAuthTokenSecret' fields.
      */
-    public async getToken(providerConfigKey: string, connectionId: string) {
-        let response = await this.getConnectionDetails(providerConfigKey, connectionId);
+    public async getToken(providerConfigKey: string, connectionId: string, instantRefresh = false) {
```

Comment:
The interface looks good to me. Let's also change this for the 2 public methods below: 
- `getRawTokenResponse`
- `getConnection`

Reply:
Thinking about it, I would maybe call this param `alwaysRefresh` or `forceRefresh`, which I think is more descriptive than `instantRefresh`.